openmetadata-service/src/main/resources/json/data/app/RetentionPolicyApplication.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "RetentionPolicyApplication",
+  "displayName": "Retention Policy",
+  "appConfiguration": {
+    "entity": "EventSubscription",
+    "retentionPeriod": "One week"
+  },
+  "appSchedule": {
+    "scheduleTimeline": "Custom",
+    "cronExpression": "0 0 * * *"
```

Comment:
This should never run daily. The default should on weekend at midnight on a weekly basis

Reply:
Addressed.